cmd/serverless/main.go

```diff
@@ -264,15 +266,18 @@ func runAgent(stopCh chan struct{}) (serverlessDaemon *daemon.Daemon, err error)
 	}()
 
 	// start appsec
-	var httpsecSubProcessor invocationlifecycle.InvocationSubProcessor
+	var (
```

Comment:
could we move all this block in an appsec specific package and call just the entrypoint here?

Reply:
Done in [5c782e7](https://github.com/DataDog/datadog-agent/pull/15378/commits/5c782e73f966af69709a01796f6c434fbb51c040)